core/core-dotspacemacs.el

```diff
@@ -129,6 +129,14 @@ several times cycle between the kill ring content.'")
 (defvar dotspacemacs-guide-key-delay 0.4
   "Guide-key delay in seconds.")
 
+(defvar dotspacemacs-which-key-delay 0.4
+  "Which-key delay in seconds.")
+
+;; TODO: Implement me
+;; Possible options should be: right bottom right-then-bottom
+(defvar dotspacemacs-which-key-position 'right-then-bottom
+  "Which-key position in seconds.")
```

Comment:
I don't think this is a good default choice, because it's a little surprising that the buffer can open in multiple places. I think bottom is the more sensible default.